src/main/java/com/adobe/cq/testing/selenium/Constants.java

```diff
@@ -53,6 +53,8 @@ private Constants() {
   public static final String PROP_CQ_CONF = "cq:conf";
   public static final String PROP_CQ_ALLOWED_TEMPLATES = "cq:allowedTemplates";
 
+  public static final String AEM_VERSION_PROPERTY_KEY = "aemVer";
 
+  public static final String AEM_VERSION_6_5 = "6.5";
```

Comment:
I'm not sure whether we should be differentiating the AEM versions. Isn't there any other way to do the differentiation?

Reply:
@absarasw @bartoszglow this is the same approach as was taken for the internal counterpart of this framework , see https://git.corp.adobe.com/CQ/selenium-it-base/pull/185.

what other way would you suggest ?